js/file.js

```diff
@@ -0,0 +1,18 @@
+function showCloseMenu() {
+  const menuSection = document.querySelector('#mobile-nav');
+  const nav = document.querySelector('#desktop-nav');
+  const headerName = document.querySelector('#logo');
+  const itemsContainer = document.querySelector('#items-container-menu');
+
+  menuSection.classList.toggle('d-none');
+  nav.classList.toggle('d-none');
+  headerName.classList.toggle('d-none');
+  headerName.classList.toggle('d-flex-center');
+  itemsContainer.classList.toggle('d-none');
+  itemsContainer.classList.toggle('d-flex');
```

Comment:
- [ ] [OPTIONAL] Part of the project requirements was 
1- `When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.` Let's make sure we implement that.
2- `When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.` you implemented that perfectly but `Contact`  goes to `About me` section let's fix that 
![image](https://user-images.githubusercontent.com/87689547/147376093-f6bbb853-7810-4f7c-af04-eb16b787cb15.png)